tests/subsys/canbus/isotp/conformance/src/main.c

```diff
@@ -451,7 +451,7 @@ static void test_send_sf_fixed(void)
 		     filter_id);
 
 	ret = isotp_send(&send_ctx, can_dev, random_data, DATA_SIZE_SF,
-			 &rx_addr_fixed, &tx_addr_fixed, send_complette_cb,
```

Comment:
They are swapped on purpose. One is the sender, the other the receiver and they are on the same device (loopback)

Reply:
Will update the bug to say this is a false positive.